sh/tests/hard-perm_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    submitted=$(ls -l "$1" && bash "$script_dirS"/student/hard-perm.sh)
```

Comment:
the test is not valid, you have to check the permission of files after the script execution

Reply:
done